src/test/java/de/gematik/vau/exception/DecryptVauMessageExceptionTest.java

```diff
@@ -129,6 +129,6 @@ void testDecryptWithAesGcmException() {
         assertThatThrownBy(() -> server.decryptVauMessage(onlyHeaderValid))
           .isInstanceOf(VauDecryptionException.class)
           .hasMessageContaining("Exception thrown whilst trying to decrypt VAU message")
-            .hasMessageContaining( "Tag mismatch!");
+            .hasMessageContaining( "Tag mismatch");
```

Comment:
Weird thing - if it is separate project then it works. If it is maven submodule it fails having "!"